### PR TITLE
File Added : Assignment 5, 6 and 9

### PR DESCRIPTION
**Pull request by shubhamdalal49**

### CODE REVIEW

File: Shubham_Dalal/JavaScript/Assignment6/Question4.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                document.getElementById("hid3").innerHTML = "Sum of all numbers factorial is :" + sum;
+            }
+        </script>
+    </head>
+    <body>
+        <div id="outside">
+            <h1>Sum of all enter factorial !!</h1>
+
+            <button onclick="fun()">Click Me</button>
+
```
Comment (Ashwin2812) [inline Shubham_Dalal/JavaScript/Assignment6/Question4.html]: Remove Extra Spaces

File: Shubham_Dalal/JavaScript/Assignment9/Count_whitespace.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="assignment9.css" />
+        <title>Document</title>
+        <script>
+            function fun() {
```
Comment (Ashwin2812) [inline Shubham_Dalal/JavaScript/Assignment9/Count_whitespace.html]: Title should be related to assignment

File: Shubham_Dalal/JavaScript/Assignment6/Question5.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="Assignment6.css" />
+        <title>Document</title>
+       <script>
```
Comment (Ashwin2812) [inline Shubham_Dalal/JavaScript/Assignment6/Question5.html]: title should be related to assignment

File: Shubham_Dalal/JavaScript/Assignment6/Question3.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="Assignment6.css" />
+        <title>Document</title>
+        <script>
+            function fun() {
+                var array = [];
```
Comment (Ashwin2812) [inline Shubham_Dalal/JavaScript/Assignment6/Question3.html]: title

File: Shubham_Dalal/JavaScript/Assignment6/Question3.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+                    }
+                    array.push(parseInt(input));
+                    add = add + array[i];
+                    document.getElementById("hid").innerHTML = "Number you have entered : "+array;
+                }
+                document.getElementById("pid").innerHTML = "the sum of numbers is :" + add;
+            }
+           
+        </script>
+    </head>
```
Comment (Ashwin2812) [inline Shubham_Dalal/JavaScript/Assignment6/Question3.html]: remove extra spaces

File: Shubham_Dalal/JavaScript/Assignment6/Question2.html [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+                document.getElementById("pid3").innerHTML = "Descending-order : " + name_arr.reverse();
+            }
+        </script>
+    </head>
+    <body>
+        <div id="outside">
+            <h1>Print name with its number:</h1>
+
+            <button onclick="fun()">Click Me</button>
+
```
Comment (Ashwin2812) [inline Shubham_Dalal/JavaScript/Assignment6/Question2.html]: remove extra spaces

File: Shubham_Dalal/JavaScript/Assignment5/even_sum_odd_product.html [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="comman_file.css" />
+        <title>Document</title>
+        <script>
+            function fun() {
+                var num,rem,sum = 0,product = 1;
```
Comment (Ashwin2812) [inline Shubham_Dalal/JavaScript/Assignment5/even_sum_odd_product.html]: title

File: Shubham_Dalal/JavaScript/Assignment5/comman_file.css [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+    margin: 20px 570px;
+}
+h1{
+    color: aliceblue;
+}
+input{
+    padding: 10px 20px;
+    border-radius: 5px;
+    display: block;
+    border-color: burlywood;
```
Comment (Ashwin2812) [inline Shubham_Dalal/JavaScript/Assignment5/comman_file.css]: add space before {

File: Shubham_Dalal/JavaScript/Assignment5/Floyd_number.html [hunk 9]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="comman_file.css">
+        <title>Document</title>
+        <script>
+            function fun() {
```
Comment (Ashwin2812) [inline Shubham_Dalal/JavaScript/Assignment5/Floyd_number.html]: title

**Review by Ashwin2812 — CHANGES_REQUESTED** [review]

File: Shubham_Dalal/JavaScript/Assignment9/prime_palindrome.html [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+    </head>
+    <body>
+        <div>
+            <h3>Check the number is Prime Palindrome or not!!!</h3>
+            <input type="number" name="" id="num" />
+            <button onclick="fun()">check prime Palindrome</button>
+            <h4 id="hid"></h4>
+        </div>
+    </body>
+
```
Comment (Pawan1005) [inline Shubham_Dalal/JavaScript/Assignment9/prime_palindrome.html]: html closing tag is missing
and remove extra lines

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Review by Ashwin2812 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Pull request merged**